src/transform/roads.py

```diff
@@ -0,0 +1,45 @@
+import sys
+
+import geopandas as gpd
+import fiona
+from jsonschema import validate
+
+from utils.yaml_api import parse_yaml
+
+GADM_FILENAME = 'gadm36_{ISO3}.gpkg'
+GADM_LAYER = 'gadm36_{ISO3}_0'
+
+
+def transform_cod():
+    transform('cod', sys.argv[1], sys.argv[2], sys.argv[3])
+
+
+def transform_gadm():
+    transform('gadm', sys.argv[1], sys.argv[2], sys.argv[3])
+
+
+def transform(source: str, input_filename: str, schema_filename: str, output_filename: str):
+    """
+    :param source: "cod" or "gadm"
+    """
+    config = parse_yaml('config.yml')
+
+    if source == "cod":
+        df_roads = gpd.read_file(f'zip://{input_filename}')
```

Comment:
For some reason there is an empty column called `'???_??????`, do you know why?

Reply:
Source schema is poorly defined. 
Source schema includes OBJECT_ID, ???_?????? and TYPE.
No road name present.
Shema mapping omitted from code.